.vscode/settings.json

```diff
@@ -39,8 +39,15 @@
   ],
```

Comment:
aside: I wonder if this should be on the `.gitignore` list?

Reply:
I go back and forth. Sometimes I commit to my user space and other times to the project. Usually I commit to the project if I already added exceptions on my local spell checker.